Cannot build from source with 1.26rc2

```
rggr.exe: can't install github.com/blampe/rreading-glasses/cmd/rggr:
# github.com/bytedance/sonic/internal/rt
go\pkg\mod\github.com\bytedance\sonic@v1.14.2\internal\rt\stubs.go:33:22: undefined: GoMapIterator
go\pkg\mod\github.com\bytedance\sonic@v1.14.2\internal\rt\stubs.go:36:54: undefined: GoMapIterator
```

--- a/go.mod
+++ b/go.mod
@@ -2,6 +2,8 @@ module github.com/blampe/rreading-glasses
 
 go 1.25.0
 
+toolchain go1.25.5
+
 tool go.uber.org/mock/mockgen
 
 tool github.com/Khan/genqlient

